pyDeltaRCM/deltaRCM_driver.py

```diff
@@ -7,6 +7,26 @@
 
 
 class pyDeltaRCM(Tools):
+    """Main pyDeltaRCM model class.
+
+    This is the main model class defined by the package. Instantiating the
+    model class is described in the :meth:`__init__` method below in detail,
+    but generally model instantiation occurs via a model run YAML
+    configuration file. These YAML configuration files define model parameters
+    which are used in the run; read more about creating input YAML
+    configuration files in the :doc:`../../guides/userguide`.
+
+    Once the model has been instantiated, the model is updated via the
+    :meth:`update`. This method coordinates the hydrology, sediment transport,
+    subsidence, and stratigraphic operation that must occur during each
+    timestep. The :meth:`update` should be called iteratively, to "run" the
+    model.
+
+    Finally, after all ``update`` steps are complete, the model should be
+    finalized (:meth:`finalize`), such that files and data are appropriately
+    closed and written to disk.
+
```

Comment:
All reads good to me, just wanted to flag that I'm not getting actual links for the methods (e.g. for :meth:`update`), again that could just be my set-up locally but wanted to point it out as something to check before merging this. 

Actually after looking at the rest of the documentation here and the docs I locally built I probably need to sort out my environment as I'm missing most of the API method documentation.

Reply:
Yeah, I'm not sure, they definitely resolve to hyperlinks for me.

![image](https://user-images.githubusercontent.com/8801322/83151332-6ebdec80-a0ca-11ea-9ba9-ce7dfeac514e.png)

Reply:
so this was a sphinx version issue, I've pinned the sphinx version to 2.0.1 for now, I think when 3.1.x comes available we should be able to add the `:recursive:` option to our `.. autosummary::` directives, and get the same behavior. 

I'll open an issue that will remind me to check back on this eventually.